package/gluon-ebtables-net-rules/files/lib/gluon/ebtables/110-net-allow-ipv6

```diff
@@ -0,0 +1,9 @@
+siteConfig = require("gluon.site_config")
+
+rule ('NET_SRC -p IPv6 --ip6-src fe80::/10 -j RETURN')
+rule ('NET_SRC -p IPv6 --ip6-dst ff00::/8 -j RETURN')
```

Comment:
Hm, yes, I think the latter makes sense. I could not think of any other protocol requiring "::" yet.

If this package is supposed to be one for strict filtering, then maybe also add something like "--ip6-src ::/128" and change the destination to "ff02::/16"?